File: JAVApb5/src/main/java/concurs/repository/IRepositoryParticipant.java
package concurs.repository;

import concurs.domain.Participant;

import java.util.List;

public interface IRepositoryParticipant {
    public Participant add(Participant e);
    public Participant delete(Participant e);
    public Participant update(Participant e);
    public Participant findOne(Long id);
    public List<Participant> findAll();
}

File: CSharp/CSharp/CSharp/repository/IRepositoryParticipant.cs
using CSharp.domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharp.repository
{
    interface IRepositoryParticipant
    {
        Participant Add(Participant e);
        Participant Delete(Participant e);
        Participant Update(Participant e);
        Participant FindOne(long id);
        List<Participant> FindAll();
    }
}
